frontend/src/components/VendorIcon.vue

```diff
@@ -17,11 +17,13 @@ limitations under the License.
 <template>
   <span>
     <img v-if="iconSrc" :src="iconSrc" :width="getWidth" :height="getHeight" :class="contentClass" style="vertical-align:middle">
-    <v-icon v-else :class="contentClass" style="font-size:1.5em">{{value}}</v-icon>
+    <v-icon v-else-if="isMdiIcon" :class="contentClass" style="font-size:1.5em">{{value}}</v-icon>
+    <v-icon v-else :class="contentClass" class="cyan--text text--darken-2" style="font-size:1.5em">mdi-blur-radial</v-icon>
```

Comment:
Do we still need the icon fallback logic for os vendor icons in the store, now that we have a fallback built-in to the VendorIcon component?

Reply:
Good point. I will remove the respective fallback

Reply:
✅